secure.c

```diff
@@ -431,7 +431,14 @@ sec_out_mcs_data(STREAM s, uint32 selected_protocol)
 	out_uint16_le(s, 0xca01);
 	out_uint16_le(s, 0xaa03);
 	out_uint32_le(s, g_keylayout);
-	out_uint32_le(s, 2600);	/* Client build. We are now 2600 compatible :-) */
+	/*
+	 * According to s.1.7 of MS-RDPESC if the build number is at least 4,304,
+	 * SCREDIR_VERSION_LONGHORN is assumed; otherwise SCREDIR_VERSIONXP is to be used
+	 */
+	/* Return the previous value of buildnum in a case of errors */
+	/* Just FYI actually */
+	//out_uint32_le(s, 2600);	/* Client build. We are now 2600 compatible :-) */
+	out_uint32_le(s, 7601);	/* Client build. We are now 7601 compatible :-) */
```

Comment:
AFAIK, no. 
I accidentally left this part after playing with chunk sizes. My fault. Sorry.

According to MS-RDPESC section 1.7 (page 15) clientBuild is used by the TS in detecting sc redirection dialect to use (SCREDIR_VERSION_LONGHORN or SCREDIR_VERSION_XP). 
I think that it has nothing to do with VC chunk size.

P.S. Sorry for long time it took to answer (was abroad and away from civilization:))

Additional info (docs links) for described (in PR)  behavior:
client side:
 3.1.4 in MS-RDPESC  (step 4)
server side:
3.2.5.2 in MS-RDPESC (handling STATUS_BUFFER_TOO_SMALL).